src/components/chats/components/Chatbox.tsx

```diff
@@ -44,6 +40,10 @@ export function Chatbox({
     [textAreaExpectedHeight, stableOnChatboxHeightChange],
   )
 
+  useEffect(() => {
+    stableOnChatboxHeightChange?.(height)
+  }, [stableOnChatboxHeightChange, height])
```

Comment:
<img width="1582" alt="image" src="https://github.com/joiahq/joia/assets/6229037/5668eef2-9344-4b94-953f-7b8b5c8cc43c">
The variable named was changed in my last merge.

Reply:
should be solved now